<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourrierManagerClass</class>
 <widget class="QDialog" name="CourrierManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1329</width>
    <height>851</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CourrierManager</string>
  </property>
  <property name="locale">
   <locale language="French" country="Belgium"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QFrame" name="toolBar">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>541</width>
     <height>751</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QGroupBox" name="grpPatient">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Info patient</string>
    </property>
    <widget class="QLabel" name="lblNom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Nom :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPrenom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Prénom :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtName">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSurname">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblDate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Date :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnToday">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Aujourd'hui</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtMonth">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtYear">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLastDate">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dernière date</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>46</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Page :</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spPage">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btnValidate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSamePatient">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Patient identique</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpFastSearch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>261</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Recherche rapide</string>
    </property>
    <widget class="QCheckBox" name="cbFastSearch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Activer la recherche rapide</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QListView" name="tablePatient">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>241</width>
       <height>231</height>
      </rect>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>630</y>
      <width>50</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>630</y>
      <width>50</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblVersion">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>670</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Version</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConfiguration">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>630</y>
      <width>50</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="serverBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>521</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Serveur</string>
    </property>
    <widget class="QLabel" name="lblTitleConnected">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Etat:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblConnected">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Non connecté</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConnect">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Reconnecter</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpOcr">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>251</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Proposition</string>
    </property>
    <widget class="QListWidget" name="ocrWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>231</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpLastMade">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>251</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Derniers courriers classés</string>
    </property>
    <widget class="QTableWidget" name="m_historyTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>231</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnRotate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>50</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLandscape">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>630</y>
      <width>50</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="topButtons">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>541</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Rechercher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Envoyer </string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpEtat">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Courrier</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>A classer:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>A envoyer :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblRemaining">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblSent">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnConvert">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>790</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Convertir</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
